sktime/transformations/series/scaledlogit.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implmenents the scaled logit tranformation."""
+
+__author__ = ["ltsaprounis"]
+__all__ = ["ScaledLogitTransformer"]
+
+from copy import deepcopy
+
+import numpy as np
+
+from sktime.transformations.base import BaseTransformer
+
+
+class ScaledLogitTransformer(BaseTransformer):
+    r"""Scaled logit transform.
+
+    Applies a scaled logit transform to the data.
```

Comment:
this should probably say also:
* requires numeric/float data
* applies the transform individually to all values (is that correct?)
* it's a logit transform, or a log transform

Reply:
Updated docs accordingly.

Reply:
? where?